persist/src/main/resources/db/migration/V3_5__AddDateAdopted.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE adopted_sites ADD COLUMN date_adopted DATE DEFAULT NULL;
```

Comment:
Could you rename this V3_6__AddDateAdopted.sql? I already migration changes and neighborhood importing with my migration script which is V3_5.

Reply:
done